crates/holochain/src/conductor/conductor.rs

```diff
@@ -2163,9 +2167,10 @@ mod misc_impls {
                 .await?;
 
             let cell = self.cell_by_id(&cell_id).await?;
-            source_chain.flush(cell.holochain_p2p_dna()).await?;
+            let actions = source_chain.flush(cell.holochain_p2p_dna()).await?;
+            assert_eq!(actions.len(), 1);
```

Comment:
Should this check just be for testing? Otherwise maybe it needs to return a user friendly error instead of a panic?